test/comparison_to_analytic_solutions.jl

```diff
@@ -24,7 +24,7 @@ e = SolidStateDetectors.elementary_charge * u"C"
         @test isapprox(C_ssd, C_true, rtol = 0.001) 
     end
 end
-=#
+
```

Comment:
I have no idea why this test fails. It passes locally on my machine...

Reply:
All of the tests calculate `C_ssd` to be 6.3529pF, while the true value `C_true` is 14.1667pF.
On my laptop, `C_ssd` has another value closer to `C_true` than the ones calculated here...
![Box_test](https://user-images.githubusercontent.com/30291312/118238015-3f99ba00-b498-11eb-85a7-3bcdff62d7dc.png)

Reply:
Hmm, strange. 
It also run rough on my system in an empty Julia environment...

Reply:
It looks like both simulations are using different initial grids and grid sizes (?)
and the grid is refined in `y` and `z` on GitHub, while it is not locally...

Locally:

![Grid_size_local](https://user-images.githubusercontent.com/30291312/118239211-c3a07180-b499-11eb-82fd-8f31237e283a.png)

GitHub:

![Grid_size_GitHub](https://user-images.githubusercontent.com/30291312/118239205-c1d6ae00-b499-11eb-8efc-76084c5777fd.png)

Reply:
On this branch, the config file format is not yet updated (#155).
Could it be that merging it to the `v0.6` branch will result in a wrong parsing of the config file somehow?

Reply:
This is indeed the source of the problem. I will have a look into it and try to fix it.

Reply:
Let's see whether merging #158 will resolve this problem.

Reply:
If I merge the `JuliaPhysics:v0.6` branch into `hervasa2:v0.6`, the tests do not fail anymore.
I don't know while the tests here still fail. But the problem should have been fixed by the PR 158.